reactjs-films-homework/src/components/MovieCard/MovieCard.tsx

```diff
@@ -24,42 +26,53 @@ export interface IGenre {
   }[];
 }
 
-const MovieCard: React.FC<ICard> = ({ props }) => {
-  const { id, title, vote_average, poster_path, genre_ids } = props;
+const MovieCard: React.FC<ICard> = ({
+  props: { id, title, vote_average, poster_path, genre_ids },
```

Comment:
you can omit 'props' here and pass all these props to components directly
`
<MovieCard id={card.id} title={card.title} ... />
`